packages/preview/splendid-mdpi/0.1.0/README.md

```diff
@@ -0,0 +1,66 @@
+# The `splendid-mdpi` Package
+<div align="center">Version 0.1.0</div>
+
+A recreation of the MDPI template shown on the typst.app homepage.
+
+## Media
+
+<p align="center">
```

Comment:
Since Typst Universe already shows the thumbnail, it's typically not replicated in the README. But that's not a hard requirement, more a matter of personal preference.

Reply:
If at all possible, I'd like to keep it as its also the readme of the repo. Perhaps typst universe can, if there exists such a file, render `universe.md` and default to `readme.md` if not? That way it becomes easier to justify enforcing a style

Reply:
You can leave it then.